public/app/features/alerting/unified/hooks/ruleGroup/useDeleteRuleFromGroup.test.tsx

```diff
@@ -111,13 +113,28 @@ describe('delete rule', () => {
   });
 
   it('should delete the entire group if no more rules are left', async () => {
-    const capture = captureRequests();
+    server.use(
```

Comment:
Can we move this to the global handlers please rather than defining just on this test? Even if it 200s for everything to begin with